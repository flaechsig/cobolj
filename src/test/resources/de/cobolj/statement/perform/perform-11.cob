IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO-WORLD.


DATA DIVISION.
   WORKING-STORAGE SECTION.
   01 WS-COUNTER PIC S9(1) VALUES 0.
   
PROCEDURE DIVISION.

  Display "Hello-000"
  Perform 001-Para.
  STOP RUN.

  001-Para.
    Display "Hello-001".
