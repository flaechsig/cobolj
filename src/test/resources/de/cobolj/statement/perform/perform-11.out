Hello-000
Hello-001
